lms/static/certificates/sass/_layouts.scss

```diff
@@ -33,9 +33,15 @@
     @extend %layout;
 
     .logo {
-        width: span(2);
+        display: block;
```

Comment:
NOTE: with the .png asset, the size of the source image is a bit greater (for better display across different devices), so this layout rule had to be adjusted some for logo display in the view's header.